.github/workflows/gource.yml

```diff
@@ -0,0 +1,32 @@
+name: Gource Action
+on:
+  push:
+    branches: [ master ]
+
+jobs: 
+  action:
+    timeout-minutes: 8
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: 'Checkout'
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    
+    - name: 'Gource Action'
+      uses: nbprojekt/gource-action@v1
+      with:
+        gource_title: 'Peirce My Heart'
+        logo_url: 'https://i.pinimg.com/736x/4e/35/fe/4e35fe64dbcd102d48e1fef43613d6e7.jpg'
+        gource_resolution: '1080p'
+        gource_fps: 60
+        gource_time_scale: 0.55
+        gource_user_scale: 0.7
+        gource_font_size: 36
+
+    - name: 'Upload gource video'
+      uses: actions/upload-artifact@v2
+      with:
+        name: Gource
+        path: ./gource/gource.mp4
```

Comment:
Is it possible to access these artifacts via links from the HTML after upload as an artifact?